test/e2e/fixture-builder.js

```diff
@@ -1691,14 +1691,6 @@ class FixtureBuilder {
           ticker: 'BNB',
           id: 'bnb-mainnet',
         },
-        'base-mainnet': {
```

Comment:
Hmm. Curious that this was here before.

Reply:
Was removing this a requirement to get tests passing? Any idea why this worked before?

Reply:
Looks like it was added originally in this PR: https://github.com/MetaMask/metamask-extension/pull/27782

Reply:
I see that this was removed as part of this PR: https://github.com/MetaMask/metamask-extension/pull/33448/files#diff-9e631c0f28d0aae4b92ffe878682c5e8479851bce192a7a4e8c6987014c96a61

But it should be safe to leave it here. I think we should revert this change, it doesn't appear necessary to backport.